designer/client/src/containers/errors/ErrorTemplate.tsx

```diff
@@ -6,17 +7,50 @@ export interface ErrorTemplateProps {
     message?: string;
 }
 
+const ErrorTemplateWrapper = styled("div")`
+    text-align: center;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+
+    h1 {
+        font-size: 96px !important;
+    }
+
+    h2 {
+        font-size: 64px !important;
+    }
+
+    .error-actions {
+        margin-top: 15px;
+        margin-bottom: 15px;
+    }
+
+    .error-actions .btn {
+        margin-right: 10px;
+    }
+`;
+
+const ErrorDetails = styled("div")`
+    font-size: 24px;
+
+    .big-blue-button {
+        margin: 45px auto;
+    }
+`;
+
 export function ErrorTemplate({ description = "", message = "", children }: PropsWithChildren<ErrorTemplateProps>): JSX.Element {
     const { t } = useTranslation();
 
     return (
-        <div className="error-template center-block">
+        <ErrorTemplateWrapper className="center-block">
```

Comment:
Do we need this 'center-block' here?

Reply:
I didn't find that classname, so I decided to leave it there.

Reply:
Me neither, that's why I'm wondering, do we need it and maybe we can just remove it?

Reply:
I think we can :)